src/components/study-manager.js

```diff
@@ -248,17 +250,60 @@ const StudyCard = ({study, onClick}) => {
 };
 
 const StudyManager = ({onStudyClick}) => {
-
+    const  userManagerHelper = new UserManagerHelper();
     const dispatch = useDispatch();
 
+    function login() {
```

Comment:
I think this whole block of code belongs somewhere else ? The studyt-manager shouldn't handle login/logout/token renew etc